client/my-sites/domains/domain-management/edit/navigation/index.jsx

```diff
@@ -0,0 +1,311 @@
+import {
+	domainAddNew,
+	domainManagementNameServers,
+	domainManagementContactsPrivacy,
+	domainManagementTransfer,
+	domainManagementDns,
+	domainManagementDomainConnectMapping,
+	domainManagementChangeSiteAddress,
+} from 'my-sites/domains/paths';
+import { emailManagement } from 'my-sites/email/paths';
+import { type as domainTypes } from 'lib/domains/constants';
+
+import './style.scss';
+
+const DomainManagementNavigationItem = function( props ) {
+	const { path, onClick, external, materialIcon, text, description } = props;
+
+	return (
+		<VerticalNavItem
+			path={ path }
+			onClick={ onClick }
+			external={ external }
+			className="navigation__nav-item"
+		>
+			<MaterialIcon icon={ materialIcon } className="navigation__icon" />
+			<div>
+				<div>{ text }</div>
+				<small>{ description }</small>
+			</div>
+		</VerticalNavItem>
+	);
+};
+
+class DomainManagementNavigation extends React.Component {
+	isDomainInNormalState() {
+		const { domain } = this.props;
+		const { expired, pendingTransfer } = domain;
+		return ! pendingTransfer && ! expired;
+	}
+
+	isDomainInGracePeriod() {
+		const { domain, moment } = this.props;
+		const { expiry } = domain;
+		return moment().subtract( 18, 'days' ) <= moment( expiry );
+	}
+
+	getEmail() {
+		const { selectedSite, translate, domain } = this.props;
+
+		if ( ! this.isDomainInNormalState() ) {
+			return null;
+		}
+
+		return (
+			<DomainManagementNavigationItem
+				path={ emailManagement( selectedSite.slug, domain.name ) }
+				materialIcon="email"
+				text={ translate( 'Email' ) }
+				description={ null && translate( 'Destination: somewhere' ) }
+			/>
+		);
+	}
+
+	getNameServers() {
+		const { selectedSite, translate, domain } = this.props;
+
+		if ( ! this.isDomainInNormalState() && ! this.isDomainInGracePeriod() ) {
+			return null;
+		}
+
+		return (
+			<DomainManagementNavigationItem
+				path={ domainManagementNameServers( selectedSite.slug, domain.name ) }
+				materialIcon="language"
+				text={ translate( 'Name servers and DNS' ) }
+				description={ null && translate( 'Destination: somewhere' ) }
+			/>
+		);
+	}
+
+	getDnsRecords() {
+		const { selectedSite, translate, domain } = this.props;
+
+		return (
+			<DomainManagementNavigationItem
+				path={ domainManagementDns( selectedSite.slug, domain.name ) }
+				materialIcon="language"
+				text={ translate( 'DNS records' ) }
+				description={ null && translate( 'Destination: somewhere' ) }
+			/>
+		);
+	}
+
+	getContactsAndPrivacy() {
+		const { selectedSite, translate, domain } = this.props;
+
+		if ( ! this.isDomainInNormalState() && ! this.isDomainInGracePeriod() ) {
+			return null;
+		}
+
+		return (
+			<DomainManagementNavigationItem
+				path={ domainManagementContactsPrivacy( selectedSite.slug, domain.name ) }
+				materialIcon="chrome_reader_mode"
+				text={ translate( 'Contacts and privacy' ) }
+				description={ null && translate( 'Privacy protection: on' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **22** times:
`translate( 'Privacy protection' )` *ES Score: 7*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/domain-status-new-navigation)